### ISSUE TITLE
[Accessibility] - Voiceover focus goes to "URL" directly after opening the popup, instead of going to first control in the popup

### ISSUE THREAD

**Issue by sryarama** [ckeditor/ckeditor4#4744]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Navigate to CKEditor demo site https://ckeditor.com/ckeditor-4/demo/
2. Activate VoiceOver from settings.
3. Click on link toolbar button.
4. Verify whether the voiceover focus is going to the first control on dialog or focus directly goes to "URL" field.

### Expected result

Voiceover focus should go to the first control present in the "Link" popup which should be in a meaningful sequence, Screen reader user should understand the sequence of the voiceover focus.

### Actual result

Voiceover focus goes to "URL" directly after opening the popup, instead of going to first control in the popup. 

### User Impact

Screen reader users face difficulty in navigation, if the Voiceover focus goes to a control which is in middle of the popup. And also visually challenged users might miss the fields above URL field.

## Other details

* Browser: Safari
* OS: iOS 14.4.2 (IPAD), VoiceOver
* CKEditor version: 4.16
* Installed CKEditor plugins: Link

**Comment by Comandeer** [comment]
This is the intended behaviour. However I must admit that it can cause some accessibility issues. Yet changing focus order will be a breaking change. So due to that I think that the reasonable solution would be to change this issue into a feature request for mechanism to control initial focus in dialogs. This way enabling the behaviour described by you would be provided as a part of [dialog definition](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_dialog_definition.html) or hidden behind a config variable.

What do you think about such a proposal?

**Comment by sryarama** [comment]
Thank for the quick reply and taking it as a feature request. And your approach seems fine to me.

Do you have any tentative date for feature availability?

Thanks,
Sreedhar

**Comment by f1ames** [comment]
Hi @sryarama, due to other priorities we don't have any ETA for this one.

**Comment by lslowikowska** [comment]
Hi @sryarama, I can see that you have posted a few issues related to accessibility already so would you please contact us at support@cksource.com? I would like to make sure we are proceeding with your requests properly. Looking forward to hearing back from you soon.
